lib/unexpected-set.js

```diff
@@ -277,23 +277,24 @@ module.exports = {
     );
 
     expect.addAssertion(
-      '<Set> to [exhaustively] satisfy <Set>',
+      '<Set> to have elements satisfying <array-like>',
```

Comment:
Starting to wonder if this wants to be "members" rather than "elements" on further thought.

Reply:
The assertions for array-like in core are called `to have items satisfying`, would make sense to match that.

Reply:
The reason I'm trying to be different is the RHS of the assertions in core are generally varargs of type <any> - perhaps we should make it vargs and then make the name the same?

Reply:
Hmm, `<array-like> to have items [exhaustively] satisfying` doesn't take varargs? https://github.com/unexpectedjs/unexpected/blob/b3273efd25125310451ba30f409bd4e0a84e6295/lib/assertions.js#L1217

Do you mean the `<assertion>` thing?

Reply:
I’m not sure what I was thinking here - might have been thrown off by the `...rest` but now realise that’s about chained assertions which we probably won’t enable.

Let me make this change & see where we end up.